index.html

```diff
@@ -1,13 +1,172 @@
+              <br>
+            </p>
+          </div>
+
+          <div class="col-12">
+            <h2 class="display-4 text-left text-wrap my-text-gray">2023.11.24(Fri) ~ 26(Sun)</h2>
+            <h3 class="text-left my-text-gray">@International Center of Conference - Bamako(Mali)</h3>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    </div>
+
+
+
+    <!-- MAIN PROGRAM-->
+    <div class="container-fluid my-bg-gray py-4" id="main-program">
+      <h3 class="text-center">
+        Main program
+      </h3>
+      <hr class="full-width mx-auto mb-5">
+      <div class="container">
+        <div id="program-cards" class="row mx-auto">
+          <!-- TO BE ADDED DYNAMICALLY-->
+          <div class="col-md-1 col-lg-1 col-xl-1 hidden-xl px-1 py-1 text-center"></div>
+
+        
+        </div>
+        
+      <!-- TO BE ADDED DYNAMICALLY-->
+      </div>
+      <p class="text-center mt-5 mb-4">
+        SEE THE WHOLE PROGRAM
+      </p>
+    </div>
     
+    <!-- SPEAKERS -->
+    <div class="container w-75 py-4 speakersContainer">
+      <h3 class="text-center">Speakers</h3>
+      <hr class="full-width mx-auto mb-5">
+      <div class="row speakers" id="speakers">
+      
+        
+      </div>
+    </div>
+  
+
+    <!--View all the participants-->
+    <a href="" type="button btn btn-block" class="more py-0 btn btn-block my-bg-light" data-toggle="collapse" data-target="#participants">
+      <img src="assets/img/icons/arrow.svg" alt="">
+    </a>
+    <div id="participants" class="collapse">
+      <div class="container-fluid space py-4">
+        <div class="container-fluid">
+          <div class="row text-center space">
+            <div class="col-12">
+              <h2>See all the participants of the Exhibition Hall</h2>
+              <hr class="full-width mx-auto mb-5">
+                          
+            </div>
+          </div>
+          
+          <div class="row text-center space" id="participantsJs">
+            
+          
+          </div>        
+        </div>
+      </div>
+    </div> <!-- check if not an extra div -->
+
+    
+
+    <!-- PARTNER-->
+    <div class="container-fluid space my-bg-gray py-4" id="partners">
+      <div class="row text-center space">
+        <div class="col-12 mb-5">
+          <h3>Partners</h3>
+          <hr class="full-width mx-auto mb-5">
+        </div>
+        
+        <div class="col-3 space" id="partner">
+          <a href=""><img class="img-thumbnail w-100 h-100" src="assets/img/logo/logo-partners/R.gif" alt=""></a>
+        </div>
+        <div class="col-3 space" id="partner">
+          <a href=""><img class="img-thumbnail w-100 h-100" src="assets/img/logo/logo-partners/LOGO_Placeholder.png" alt=""></a>
+        </div>
+        <div class="col-3 space" id="partner">
+          <a href=""><img class="img-thumbnail w-100 h-100" src="assets/img/logo/logo-partners/R.gif" alt=""></a>
+        </div>
+        <div class="col-3 space" id="partner">
```

Comment:
The partners section should not be visible in the home page of the mobile screen. It would be nice it you fixed this issue.

This is your design:
![partners ](https://user-images.githubusercontent.com/79506105/235284500-955778a0-9982-4b3e-9c8b-2c3c0f2ef27e.png)


Please follow design requirements your design  should look like this in you home page:

![Screenshot 2023-04-29 at 7 08 21 AM](https://user-images.githubusercontent.com/79506105/235284570-d10f0fb2-d994-4f3b-a3c4-95edcc993f33.png)


Please Bare in mind that the partnes section should be visible in the about section of your mobile screen Just like in the design.

For a better understanding please follow the following requirements and guidelines:

Page Structure and Navigation

The project contains a Home page and About page.
It's possible to navigate between the pages, there's a link from Home page to About page and the opposite too.
Both pages contain the same navbar and footer.


Navbar

Mobile Size

It contains an hamburger menu on the left side.
It's background is transparent or the same color/pattern as the top section background.


Desktop Size

It is divided in a top dark bar and a bottom white bar.
The top short bar section with social media links, logout, and language links aligned to the right.
The taller bottom bar with a logo and a list of page links.
The top bar contains a logo to the left.
The top bar contains a list of page links aligned to the right.


Footer

It contains a logo aligned to the left and text aligned to the right.


Mobile Size

It has a white background.


Desktop Size

About Page: It has a dark background.
Home Page: It has a white background.


Home Page

It contains a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text.
It contains a middle section with a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom.
It contains a bottom section with a column that includes a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description.


Mobile Size

The middle section article list is organized in a vertical column.
The middle section individual articles have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right.
The bottom section articles are organized in a vertical column.


Desktop Size

The middle section article list is organized in an horizontal row.
The middle section individual articles have their elements organized in an vertical column, with the image at the top, title in the center, and description at the bottom.
The bottom section articles are organized in two columns of three articles each.


About Page

It contains a top section with a centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it.
It contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.
It contains a bottom section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them.


Mobile Size

The bottom section images are organized in a vertical column.


Desktop Size

The bottom section images are organized in an horizontal row.


Design and Styling

The pages should look almost identical to the original design, outside the content differences.
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.


Basic color swatch: #272A31, #EC5242, #D3D3D3


The pages use the same fonts or simialr fonts from the original design.


Fonts used originally: Cocogoose, Lato


All page elements have similar spacing, borders, shadows, and paddings than the orignal design.


HTML code

The HTML uses semantic tags, sections use section, articles use article, the nav and footer use nav and footer.
There's at least 1, but no more than 1, h1 tag in each page.
The HTML markup is well structured and follows a clear hierarchy, with sections, and articles, each containing appropriated Hx headers and paragraphs.
The page is accessible, images contain alt attributes and there's not images or elements blocking the text in both screen sizes.
The logo contains a hyperlink to the homepage.


CSS code

Selector classes are used multiple times and rules are not repeated, the code is DRY.
The CSS styling is built as mobile-first, media queries are used to change the design from mobile to desktop, not the other way around.
Media queries are properly organized at the bottom of the stylesheet.
NOTE: Students can use CSS frameworks (Bootstrap, Tailwind CSS and any other CSS framework) for styling (if they want to).


Note: It is fine if the student has used Bootstrap or any other framework.


JavaScript code

Dynamically generated HTML

The section "Featured speakers" is generated dynamically with JavaScript.
The JavaScript code is within it's own .js file, not inside the HTML in scripttags.
The list of speakers used to generate the HTML is assigned to a variable as an array or object.
The list of speakers contain an speaker object for each that holds their specific data.


// You can use an array to store the objects
const speakerList = [ { name: "Speaker Name",... } ]
// It's ok to use an object too
const speakerList = { speaker1: { name: "Speaker Name",... } }

The object contains all the information needed to generate the dynamic element, at least the name, role, description, and image url about the speaker. For example:


{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}

There is a function that iterates through the speakers list on window load and creates HTML elements with the data from each object. For example:


window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })

Reply:
Fixed @KayLemba, thanks for the review.